pytket/docs/requirements.txt

```
@@ -1,6 +1,6 @@
 sphinx >= 4.5
 sphinx_autodoc_annotation >= 1.0
-sphinx_book_theme ~= 0.3.3
+sphinx_book_theme >= 1.0
```

```suggestion
sphinx_book_theme ~= 1.0
```
Probably better so that we don't pick up 2.0 accidentally.